### PR TITLE
updating change log

### PR DESCRIPTION
**Pull request by ashishsingh18**
updating change log

### CODE REVIEW

File: 7_ReleaseNotes.txt [hunk 1]
```diff
@@ -18,6 +18,13 @@
   - Recurrence fixes
   - Pseudoprogression fixes
 - Generic GUI bug fixes and improvements 
+- Apply mask functionaity from drawing panel added
+- bvec reorientation functionality added from Utilities
```
Comment (sarthakpati) [inline 7_ReleaseNotes.txt:22]: I believe it should be **BVEC**

File: 7_ReleaseNotes.txt [hunk 2]
```diff
@@ -18,6 +18,13 @@
...
+- Headers added to csv features files for multiple applications
+- Fixed application font mismatch on GUI
+- Fixed mask opacity change functionaity from comparison mode
+- Fixes for issues identified during Testathon Feb 2020
```
Comment (sarthakpati) [inline 7_ReleaseNotes.txt:26]: Remove this, as it has got nothing to do with the code and getting pertinent information will involve going to GitHub and searching for relevant tag.

File: 7_ReleaseNotes.txt (hunk 1 above)
Comment (ashishsingh18) [reply 7_ReleaseNotes.txt]: Done and moved under Utilities category.

File: 7_ReleaseNotes.txt (hunk 2 above)
Comment (ashishsingh18) [reply 7_ReleaseNotes.txt]: removed testathon part. Is it better now?

**Comment by ashishsingh18** [comment]
@CBICA/devteam This is ready for review

**Pull request merged**